Package.swift

```diff
@@ -176,6 +176,10 @@ let package = Package(
       url: "https://github.com/SlaunchaMan/GCDWebServer.git",
       revision: "935e2736044e71e5341663c3cc9a335ba6867a2b"
     ),
+    .package(
+      url: "https://github.com/google/interop-ios-for-google-sdks.git",
+      "100.0.0" ..< "101.0.0" , branch: "main"
```

Comment:
Only one of these can be specified. The tag has been made so delete `, branch: "main"` https://github.com/google/interop-ios-for-google-sdks/tags